characters/electro/beidou.md

```diff
@@ -92,6 +92,7 @@ Beidou creates a **Thunderbeast's Targe** around herself, dealing Electro DMG to
 * Increases active character’s resistance to interruption and decreases DMG taken.
 * A maximum of 1 lightning **discharge** can be triggered per second.
 * Snapshot on cast.
+  * However, if you swap to Beidou and cast her Burst too quick (Left Alt + Switch Character), her Burst might fail to snapshot.
```

Comment:
Not sure this is what it shows. I think it instead shows that the Lightning damage snapshots on cast, but the initial skill hit is calculated on hit.

Reply:
fixed the wording for clarity